demisto_sdk/commands/format/update_generic_yml.py

```diff
@@ -144,6 +144,16 @@ def copy_tests_from_old_file(self):
             if not self.data.get("tests", "") and self.old_file.get("tests", ""):
                 self.data["tests"] = self.old_file["tests"]
 
+    def section_order_to_lower(self):
+        """
+        Converts 'sectionOrder' field to lowercase 'sectionorder' and removes the original camelCase field.
+        This ensures compliance with the expected lowercase field naming convention.
+        """
+        if "sectionOrder" in self.data:
```

Comment:
Might be better to catch a general string of "sectionorder" with lowercase letters or camelcase letters that could be everywhere in the string, and convert it to all lowercase. 
Meaning catching not only the "sectionOrder" case, but also "SectionOrder" and "Sectionorder" as well.